pandas/io/formats/style.py

```diff
@@ -204,6 +206,101 @@ def _repr_html_(self) -> str:
         """
         return self.render()
 
+    def set_tooltips(self, ttips: DataFrame) -> "Styler":
+        """
+        Add string based tooltips that will appear in the `Styler` HTML result. These
```

Comment:
versionadded 1.2

Reply:
done, except I guess this is 1.3.0 now? Sorry, wasn't available over the holidays so couldn't get to this.

Reply:
yes

Reply:
no problem at all. thanks for the patch.